src/components/pages/Profile/Header/VerifiedPatch.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from 'gatsby'
+import React, { ReactElement } from 'react'
+import { ReactComponent as VerifiedPatchIcon } from '../../../../images/patch_check.svg'
+import { useProfile } from '../../../../providers/Profile'
+import Tooltip from '../../../atoms/Tooltip'
+import styles from './VerifiedPatch.module.css'
+
+export default function VerifiedPatch(): ReactElement {
+  const tooltip = (
+    <span>
+      This member has been officially onboarded to the GEN-X network.{' '}
+      <Link to="#">Learn more</Link>
```

Comment:
Question (blocking): it looks like there is a dead link here, where is the `Learn More` supposed to point to?

Reply:
Good catch. I think we can remove this for now, unless we have a fitting section in the docs already @oceanByte @Abrom8